Cargo.toml

```diff
@@ -14,13 +14,13 @@ bitcoin = { version = "0.25", features = ["use-serde"] }
 bitcoincore-rpc-json = "0.12"
 futures = "0.3.5"
 hex = "0.4.2"
-jsonrpc_client = { git = "https://github.com/thomaseizinger/rust-jsonrpc-client", rev = "c7010817e0f86ab24b3dc10d6bb0463faa0aace4", features = ["reqwest"] }
-reqwest = { version = "0.10", default-features = false, features = ["json", "native-tls"] }
+jsonrpc_client = { git = "https://github.com/thomaseizinger/rust-jsonrpc-client", rev = "f60c839481c1ac68909ada0141a3a3bf085bb1af", features = ["reqwest"] }
```

Comment:
I've released the lib to crates.io a couple of days ago :)

You can just depend on that instead of the Git rev!

Reply:
hmm release 0.4 doesn't not build because it is using request 0.10. 0.11 is needed.
```the trait bound `reqwest::Client: jsonrpc_client::SendRequest` is not satisfied
  --> src/bitcoind_rpc.rs:18:1
   |
18 | #[jsonrpc_client::implement(BitcoindRpcApi)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `jsonrpc_client::SendRequest` is not implemented for `reqwest::Client`
   |
   = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)```

Reply:
Well, use release 0.5 then? :)

That one depends on reqwest 0.11

Reply:
https://docs.rs/jsonrpc_client/0.5.0/jsonrpc_client/

Reply:
did not know/see there was a 0.5 release.